suite-native/module-passphrase/src/screens/PassphraseEmptyWalletScreen.tsx

```diff
@@ -0,0 +1,98 @@
+
+    const [isSheetVisible, setIsSheetVisible] = useState(false);
+
+    const handleOpenEmptyWallet = () => {
+        setIsSheetVisible(true);
+    };
+
+    return (
+        <Screen screenHeader={<PassphraseScreenHeader />}>
+            <VStack spacing="large">
+                <Text variant="titleMedium" style={applyStyle(textStyle, { widthPercentage: 90 })}>
+                    <Translation id="modulePassphrase.emptyPassphraseWallet.title" />
+                </Text>
+                <Card style={applyStyle(cardStyle)}>
+                    <VStack
+                        alignItems="center"
+                        spacing="medium"
+                        style={applyStyle(cardContentStyle)}
+                    >
+                        <Image
+                            source={require('../assets/questionMarks.png')}
+                            style={applyStyle(imageStyle)}
+                        />
+                        <Text variant="highlight" textAlign="center">
+                            <Translation id="modulePassphrase.emptyPassphraseWallet.confirmCard.description" />
+                        </Text>
+                    </VStack>
+                    <Button onPress={handleOpenEmptyWallet}>
+                        <Translation id="modulePassphrase.emptyPassphraseWallet.confirmCard.button" />
+                    </Button>
+                </Card>
+                {/* TODO https://github.com/trezor/trezor-suite/issues/12207 */}
+                <Text>OR</Text>
+                <VStack spacing={20}>
+                    <VStack alignItems="center" spacing="extraSmall">
+                        <Text
+                            textAlign="center"
+                            variant="highlight"
+                            style={applyStyle(textStyle, { widthPercentage: 80 })}
+                        >
+                            <Translation id="modulePassphrase.emptyPassphraseWallet.expectingPassphraseWallet.title" />
+                        </Text>
+                        <Text
+                            variant="hint"
+                            color="textSubdued"
+                            textAlign="center"
+                            style={applyStyle(textStyle, { widthPercentage: 70 })}
+                        >
+                            <Translation id="modulePassphrase.emptyPassphraseWallet.expectingPassphraseWallet.description" />
+                        </Text>
+                    </VStack>
+                    <Box style={applyStyle(buttonWrapperStyle)}>
+                        <Button colorScheme="tertiaryElevation0">
+                            <Translation id="modulePassphrase.emptyPassphraseWallet.expectingPassphraseWallet.button" />
+                        </Button>
+                    </Box>
+                </VStack>
+            </VStack>
+            {isSheetVisible && <EmptyWalletInfoSheet onClose={() => setIsSheetVisible(false)} />}
```

Comment:
Please add isVisible prop to `EmptyWalletInfoSheet`
```suggestion
             <EmptyWalletInfoSheet isVisible={isSheetVisible} onClose={() => setIsSheetVisible(false)} />
```

and the pass it to the underlying BottomSheet component. So the use of BottomSheet api is consistent across the codebase.

Reply:
nit: do not use inline function.

Reply:
https://github.com/trezor/trezor-suite/pull/12193/commits/fd61a43929e4a65804e39a40a5c1d039432da8cf